parm/config/gfs/config.aeroanl

```diff
@@ -20,4 +19,12 @@ export io_layout_y=@IO_LAYOUT_Y@
 
 export JEDIEXE=${HOMEgfs}/exec/fv3jedi_var.x
 
+if [[ "${DOIAU}" == "YES" ]]; then
+  export restart_interval="3"
+  export AEROVARYAML=${HOMEgfs}/sorc/gdas.cd/parm/aero/variational/3dvar_fgat_gfs_aero.yaml
```

Comment:
I will work on linking these things to the workflow root directory in a subsequent PR.